contracts/EasyAuction.sol

```diff
@@ -176,15 +176,13 @@ contract EasyAuction is Ownable {
             "minimumBiddingAmountPerOrder is not allowed to be zero"
         );
         require(
-            orderCancelationPeriodDuration <= duration,
+            orderCancellationEndDate <= auctionEndDate,
             "time periods are not configured correctly"
         );
+        require(auctionEndDate != 0, "auction end date cannot be zero");
```

Comment:
I would prefer checking that orderCancellationEndDate is in the future, i.e. > block.timestamp